### ISSUE TITLE
WriteOnlyProperty false positive when used in new

### ISSUE THREAD

**Issue by VasekPurchart** [slevomat/coding-standard#334]
When applying the `SlevomatCodingStandard.ControlStructures.NewWithParentheses` to some old code I started to get `WriteOnlyProperty` errors. Upon further investigation it seems, that `WriteOnlyProperty` will take this:

```php
$foo = new $this->fooClass;
```

as a "read usage", but not like this:

```php
$foo = new $this->fooClass();
```

I know neither one is pretty, but I think it should behave consistently.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/eb3037b89c54fd6b178adbea51f38d10c7feea78

**Comment by VasekPurchart** [comment]
Thanks, tested on master :)
